src/service/http/fetcher.ts

```diff
@@ -13,6 +13,15 @@ export const axiosIns = axios.create({
   data: null,
 })
 
+export const v2AxiosIns = axios.create({
```

Comment:
We can use response interceptors to return `response.data`.
Then we don't need to get results by `data.data` when using `v2AxiosIns`.